behind-the-veil-siteroot/imports/ui/components/booking/BookingCalendarView.jsx

```diff
@@ -136,12 +136,21 @@ const BookingCalendarView = ({ bookingsData }) => {
         events={events}
         eventPropGetter={(myEventsList) => {
           let backgroundColor;
+          let className;
```

Comment:
This className variable is unused. Also, was there any way that we could use these tailwind colour variables instead of hard coding the colour hex codes?

<img width="454" alt="Screenshot 2024-08-19 at 10 17 15 AM" src="https://github.com/user-attachments/assets/7382d1c6-d884-4ca9-b16c-157e299b089e">

Just b/c if we ever change the tailwind colour configs, then they would also have to be manually updated here